From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matthijs Velsink <mvelsink@gnome.org>
Date: Tue, 12 Mar 2024 21:30:31 +0100
Subject: [PATCH] subprojects: Update gvc to latest commit

A crash possibility has been fixed in libgnome-volume-control!25.

A similar crash was reported on the Settings side, so update to the
latest gvc commit to hopefully prevent further crashes.

Related: gnome-shell#7173
Related: #2924
---
 subprojects/gvc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/subprojects/gvc b/subprojects/gvc
index dbfbacc9571f..91f3f4149066 160000
--- a/subprojects/gvc
+++ b/subprojects/gvc
@@ -1 +1 @@
-Subproject commit dbfbacc9571fade87855907b78c6ed5e27c910dd
+Subproject commit 91f3f41490666a526ed78af744507d7ee1134323
